doc/tutorials/using_celadon_as_uos.rst

```diff
@@ -0,0 +1,113 @@
+.. _using_celadon_as_uos:
+
+Using Celadon as User OS
+########################
+
+`Celadon <https://01.org/projectceladon/>`_ is an open source Android* software reference stack
+for Intel architecture. It builds upon a vanilla Android stack and incorporates open sourced components
+that are optimized for the hardware. This tutorial describes how to run Celadon as the User OS
+on the ACRN hypervisor. We are using Kaby Lake-based NUC (model NUC7i7DNHE) in this tutorial.
+Other :ref:`ACRN supported platforms <hardware>` should work as well.
+
+Introduction to OVMF
+********************
+
+Open Virtual Machine Firmware (OVMF) is an `EDK II
+<https://github.com/tianocore/tianocore.github.io/wiki/EDK-II>`_
+based project to enable `UEFI <https://github.com/tianocore/tianocore.github.io/wiki/UEFI>`_
+support for Virtual Machines. OVMF contains sample UEFI firmware for `QEMU <http://www.qemu-project.org/>`_ and `KVM
+<http://www.linux-kvm.org/>`_.
+
+Prerequisites
+*************
+
+* Ubuntu 16.04 or higher with at least 150G free disk space
+* Intel NUC Kit ( Reference Platforms: :ref:`ACRN supported platforms <hardware>` )
+* BIOS version should be 0057 or later, the ``Device Mode`` option is selected
+  for the internal USB 3.0 port header on the USB category of the Devices tab
+* If you need to use ``adb`` or ``fastboot`` tool with a Type-A to Type-A USB cable
+  for the Celadon User VM, then you must provide a USB dongle (e.g. `Dawson Canyon USB 3.0 female
+  to 10-pin header cable <https://www.gorite.com/dawson-canyon-usb-3-0-female-to-10-pin-header-cable>`_)
+  to connect to the internal USB 3.0 port header on the main board of the Intel NUC system.
+  Refer to the `Technical Product Specification
+  <https://www.intel.com/content/dam/support/us/en/documents/mini-pcs/nuc-kits/NUC7i5DN_TechProdSpec.pdf>`_
+  to identify the USB 3.0 port header on the main board.
+* Two HDMI monitors
+
+Build Celadon from source
+*************************
+
+#. Follow the `Build Celadon from source instructions
+   <https://01.org/projectceladon/documentation/getting_started/build-source>`_
+   to get the Celadon project source code ready.
+
+#. Select Celadon build target::
+
+      $ cd <Celadon project directory>
+      $ source build/envsetup.sh
+      $ lunch cel_apl-userdebug
+
+   .. note:: You can run ``lunch`` with no arguments to manually choose your Celadon build variants.
+
+#. Download these additional patches and apply each one individually with the command::
```

Comment:
When do you expect these to be merged?

It may be good to indicate which revision this tutorial is based on (and which needs all these patches). So the reader can more easily track down if a newer version has it already or not.

Reply:
OK, this is good to know. Thanks @ttzeng